client/src/views/Home.vue

```diff
@@ -8,6 +7,11 @@ import RunPipelineMenu from '@/components/RunPipelineMenu.vue';
 import Upload from '@/components/Upload.vue';
 import NavigationBar from '@/components/NavigationBar.vue';
 import { getPathFromLocation, getLocationFromRoute } from '@/utils';
+import {
+  mapActions,
+  mapMutations,
+  mapGetters,
+} from 'vuex';
```

Comment:
Not a blocker, but just a note: Generally speaking, third party module imports are separated from internal imports, with third party imports usually going above internal ones.

Reply:
Actually, that should be a lint error.  I believe the `@` tricks eslint into believing everything is an external import.  I'll have to look into that, but please hoist this above the internal imports.

Reply:
should I do this for Upload.vue too?

Reply:
Looks like it's fixed in 9fb16c7